client/test/app/hearings/components/dailyDocket/DailyDocketGuestLinkSection.test.js

```diff
@@ -1,78 +1,53 @@
 import React from 'react';
-import { screen, render } from '@testing-library/react';
-import { axe } from 'jest-axe';
+import { render, screen } from '@testing-library/react';
 // eslint-disable-next-line max-len
 import { DailyDocketGuestLinkSection } from '../../../../../app/hearings/components/dailyDocket/DailyDocketGuestLinkSection';
```

Comment:
```suggestion
import {
  DailyDocketGuestLinkSection
} from 'app/hearings/components/dailyDocket/DailyDocketGuestLinkSection';
```

`app` is [registered as a alias](https://github.com/department-of-veterans-affairs/caseflow/blob/fd2f331f6719fcd1838a1f444d65bd951bf0e257/client/webpack.config.js#L44) that can be used to remove the need for relative pathing.